actors/reward/src/types.rs

```diff
@@ -3,8 +3,15 @@
 
 use fvm_ipld_encoding::tuple::*;
 use fvm_shared::address::Address;
+use fvm_shared::bigint::bigint_ser::BigIntDe;
 use fvm_shared::econ::TokenAmount;
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
+pub struct ConstructorParams {
+    pub power: Option<BigIntDe>,
```

Comment:
Could you try out using
```
    #[serde(with = "bigint_ser")]
    pub power: Option<StoragePower>,
```
now that this is within a struct? It might not work because of the `Option` though.